ydb/library/yql/udfs/common/knn/knn-serializer.h

```diff
@@ -12,21 +12,21 @@
 using namespace NYql;
 using namespace NYql::NUdf;
 
-template <typename T, EFormat Format>
+template <typename To>
```

Comment:
`To` is a bit confusing due to style guide.
All types should have a prefix `T`.
There should be better to leave as it was: `typename T`